broker/volumebroker/server/volume.go

```diff
@@ -82,6 +83,16 @@ func (s *Server) convertOnmetalVolumeResources(resources corev1alpha1.ResourceLi
 	}, nil
 }
 
+func (s *Server) getOnmetalEncryption(volume *AggregateOnmetalVolume) *ori.EncryptionSpec {
```

Comment:
This function should be called `convertOnmetalVolumeEncryption` as e.g. `convertOnmetalVolumeAccess`.